### PR TITLE
docs: update docs with up-to-date information

### PR DESCRIPTION
**Pull request by RKNF404**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,15 @@ Official support is only provided via [secureblue](https://github.com/secureblue
...
+\
+Additionally, the following flags are available that provide extra hardening but may cause breakage or usability issues:
 
-- `chrome://flags/#enable-network-service-sandbox`
 - `chrome://flags/#show-punycode-domains`
 - `chrome://flags/#disable-cross-origin-referrers`
 
-The following flags are also provided for convenience if you need them:
-
-- `chrome://flags/#incognito-launch`
```
Comment (RoyalOughtness) [inline README.md:34]: this one is still present

File: README.md [hunk 2]
```diff
@@ -23,16 +23,15 @@ Official support is only provided via [secureblue](https://github.com/secureblue
...
-- `chrome://flags/#extensions-support`
```
Comment (RoyalOughtness) [inline README.md:35]: also `chrome://flags/#strict-popup-blocking` was added

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -48,10 +48,23 @@ We will then take care of the issue as soon as possible.
...
 ### Building locally
 #### Setup
-Download and run copr_script.sh:
+Enable the COPR repository (as root):
 
-`wget https://raw.githubusercontent.com/secureblue/hardened-chromium/master/copr_script.sh`
-`./copr_script.sh`
+`dnf copr enable secureblue/hardened-chromium`
+
+Install the `chromium-clean-source` package (as root):
```
Comment (RoyalOughtness) [inline CONTRIBUTING.md:59]: (or in a distrobox)

File: README.md (hunk 2 above)
Comment (RKNF404) [reply README.md:35]: It's enabled by default, so it falls under compat rather than hardening.

File: CONTRIBUTING.md (hunk 3 above)
Comment (RKNF404) [reply CONTRIBUTING.md:59]: Not sure how to set that up personally.

Comment (RKNF404) [reply CONTRIBUTING.md:59]: Oh, like install the package in a Distrobox?

Comment (RoyalOughtness) [reply CONTRIBUTING.md:59]: yes, just meant the comment could recommend distrobox as an alternative

File: README.md (hunk 2 above)
Comment (RoyalOughtness) [reply README.md:35]: Right but shouldn't we still list them here?

File: CONTRIBUTING.md (hunk 3 above)
Comment (RKNF404) [reply CONTRIBUTING.md:59]: the copr script accounts for a very specific location of the archive, unless you mean doing all of the steps in the distrobox?

Comment (RoyalOughtness) [reply CONTRIBUTING.md:59]: yeah that's what i meant, like using a distrobox as a hardened-chromium devenv

File: README.md (hunk 2 above)
Comment (RKNF404) [reply README.md:35]: They are intended to be fixes for sorta niche features or edge cases, and there are quite a few to list.

File: CONTRIBUTING.md (hunk 3 above)
Comment (RKNF404) [reply CONTRIBUTING.md:59]: I added a note about using a container, like distrobox

**Review by RoyalOughtness — APPROVED** [review]

**Pull request merged**